lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -583,7 +583,7 @@ def verify_flow_rules_deletion(self):
         print("Checking if all uplink/downlink flows were deleted")
         dpath = get_datapath()
         flows = get_flows(
-            dpath, {"table_id": self.SPGW_TABLE, "priority": 0}
+            dpath, {"table_id": self.SPGW_TABLE}
         )
         assert(
             len(flows) == 2), "There should only be 2 default table 0 flows"
```

Comment:
this should be 1 after ipfix changes, no?

Reply:
hm no, the ipfix flow is still there, just with higher pri

Reply:
ah, okay. for a moment I thought they were removed as well.